pymemcache/client/hash.py

```diff
@@ -130,6 +130,12 @@ def _get_client(self, key):
                         self._last_dead_check_time = current_time
 
         server = self.hasher.get_node(key)
+        # We've ran out of servers to try
+        if server is None:
+            if self.ignore_exc is True:
```

Comment:
@renstrom I always do that unless there is a reason to support all falsy operations, i.e do you think this should work:

```python
self.ignore_exc = 1
```